packages/actions/src/Commands/Aliases/MakeExporterCommand.php

```diff
@@ -3,7 +3,9 @@
 namespace Filament\Actions\Commands\Aliases;
 
 use Filament\Actions\Commands;
+use Symfony\Component\Console\Attribute\AsCommand;
 
+#[AsCommand(name: 'make:filament-exporter', aliases: ['filament:exporter'])]
```

Comment:
For the alias command classes, do the arguments need to be swapped? So `make:filament-exporter` is an alias of `filament:exporter`? Maybe I am understanding it wrong, but Symfony's docs do not explain this

Reply:
Maybe we dont need `aliases: ` at all yet since we aren't using the `$aliases` property on the commands? These could be treated as separate commands where the alias classes just reference their own name?

Then we refactor to `$aliases` and get rid of these classes in v4

Reply:
Yes, I think we should treat these alias as separate command. I will go ahead and do that.

Reply:
> For the alias command classes, do the arguments need to be swapped? So `make:filament-exporter` is an alias of `filament:exporter`? Maybe I am understanding it wrong, but Symfony's docs do not explain this

For swapping of the arguments, I don't think so but I will test it out and see where that leads me.